Assets/script.js

```diff
@@ -0,0 +1,115 @@
+speakers=[
```

Comment:
kindly, I suggest you follow a pattern in the format of the images, or only use jpg or base64, do not mix, personally I recommend you use only JPG is much more friendly to see. ;-)